packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -2,7 +2,7 @@ annotations:
   catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: CIS Benchmark
-  catalog.cattle.io/kube-version: '>= 1.21.0-0 < 1.25.0-0'
+  catalog.cattle.io/kube-version: '>= 1.21.0-0 < 1.26.0-0'
```

Comment:
Is there a reason why we are hardcoding to not work on 1.26? It seems like an extra hassle for us to maintain? Note: I didn't check this across all the charts, but wanted to ask if there was a specific reason. I noticed it in the SUSE projects, so was curious.

Reply:
No specific reason! I just didn't check if this chart has any deprecations that need to be addressed. More around erring on the side of caution.

Reply:
@deniseschannon generally we set the Rancher version annotation to versions that QA has explicitly tested during a release, so charts should always be setting an upper bound here as a best practice. 

Otherwise, if a chart wouldn't work in a new Kubernetes version, we'd have to go back and OOB release previous charts to no longer be deployable on the new cluster version to inform users.